bayesian/api_v1.py

```diff
@@ -704,7 +704,7 @@ def post():
                 filepath = filepaths[index]
                 content = manifest_file_raw.read().decode('utf-8')
             # For npm-list.json, we need not verify it as its not going to mercator task
```

Comment:
Change this comment as we support more than npm-list.json files

Reply:
done